posts/2021-04-12-async-vision-doc-shiny-future.md

```diff
@@ -0,0 +1,85 @@
+
+The "shiny future" stories describe those some characters achieving those same goals, but looking forward a few years into the future. They are meant to illustrate the experience we are aiming towards, and to give the overall context for the RFCs and other kinds of changes we want to pursue.
+
+#### What happens when there are multiple "shiny future" stories about the same thing?
+
+During this brainstorming period, we want to focus on getting as many ideas as we can. Having multiple "shiny futures" that address the same problem is a feature, not a bug, as it will let us mix-and-match later to try and find the best overall plan.
+
+#### Do we have to know exactly how we will achieve the "shiny future"?
+
+No! Of course, we will eventually have to figure out the precise designs, but at this point we're more interested in talking about the experience we aim to create. That said, shiny future PRs can also include [design docs] that give technical details.
+
+#### What if we write a "shiny future" story but it turns out to be impossible to implement?
+
+Glad you asked! The vision document is a living document, and we intend to revisit it regularly. This is important because it turns out that predicting the future is hard. We fully expect that some aspects of the "shiny future" stories we write are going to be wrong, sometimes very wrong. We will be regularly returning to the vision document to check how things are going and adjust our trajectory appropriately.
+
+#### What if I'm not sure how to get started?
+
+[Ryan Levick] and [I] have been hosting "vision doc writing sessions" and they expect to continue doing this. These sessions are a fun way to work with other people from the Rust community. Keep an eye on the [babysteps blog] or twitter for announcements.
+
+#### Can I still submit "status quo" stories?
+
+Yes! We are accepting status quo stories right up until the end of the brainstorming period.
+
+#### What is the "scope" of a shiny future story?
+
+All the stories in the vision doc are meant to cover the full "end to end" experience of using async Rust. That means that sometimes they will take about things that are really part of projects that are outside of the Rust org. For example, we might write a shiny future that involves how the standard library has published standard traits for core concepts and those concepts have been adopted by libraries throughout the ecosystem.
+
+#### How long does the brainstorming period last, and what happens next?
+
+The brainstorming period lasts until the end of the month. After that, the [working group leads] are going to merge the remaining stories and get to work drafting a synthesized vision document that incorporates elements of the various stories that have been submitted (the ["harmonizing"] period).
```

Comment:
feels a little like it's missing a conclusion- depending on the reaction to the previous comment, this may be moot or potentially i'd recommend changing this question from a question to a closing paragraph, but curious what you think.